dev/breeze/src/airflow_breeze/commands/developer_commands.py

```diff
@@ -640,6 +640,13 @@ def build_docs(
             for directory in docs_dir.rglob(dir_name):
                 get_console().print(f"[info]Removing {directory}")
                 shutil.rmtree(directory, ignore_errors=True)
+    package_list_from_env = os.getenv("PACKAGE_LIST")
+    if package_list_from_env:
+        get_console().print(
+            f"\n[info]Populating provider list from PACKAGE_LIST env as {package_list_from_env}"
```

Comment:
NIT: when we see that someone specifies packages also as arguments (if doc_packages) then we should print warning that we are overriding the packages specified as argument. INFO is not enough in this case.

Reply:
Alternatively, we could merge those two lists, but I think overriding is also a good choice (if we see warning).

Reply:
Merging is an overkill I feel, we should just mention overriding in a warning.

Reply:
Yep.